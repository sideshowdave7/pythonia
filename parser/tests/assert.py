id = 5
assert id in [1, 2, 3, 4, 5, 6], "id is not in the list!: %r" % id
