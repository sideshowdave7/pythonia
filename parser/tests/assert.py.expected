(program
 (= (id) 5)
 (assert
  (comparison id ("in" (list 1 2 3 4 5 6)))
  (term "id is not in the list!: %r" ("%" id))))

